cpp/benchmarks/common/generate_input.cu

```diff
@@ -409,61 +505,43 @@ void append_string(Char_gen& char_gen, bool valid, uint32_t length, string_colum
  */
 template <>
 std::unique_ptr<cudf::column> create_random_column<cudf::string_view>(data_profile const& profile,
-                                                                      std::mt19937& engine,
+                                                                      thrust::minstd_rand& engine,
                                                                       cudf::size_type num_rows)
 {
-  auto char_dist = [&engine,  // range 32-127 is ASCII; 127-136 will be multi-byte UTF-8
-                    dist = std::uniform_int_distribution<unsigned char>{32, 137}]() mutable {
-    return dist(engine);
-  };
-  auto len_dist =
-    random_value_fn<uint32_t>{profile.get_distribution_params<cudf::string_view>().length_params};
-  auto valid_dist = std::bernoulli_distribution{1. - profile.get_null_frequency()};
-
-  auto const avg_string_len = non_fixed_width_size<cudf::string_view>(profile);
-  auto const cardinality    = std::min(profile.get_cardinality(), num_rows);
-  string_column_data samples(cardinality, cardinality * avg_string_len);
-  for (cudf::size_type si = 0; si < cardinality; ++si) {
-    append_string(char_dist, valid_dist(engine), len_dist(engine), samples);
-  }
-
+  // ~10% UTF-8. ~90% ASCII.
+  // ~20% space, ~80% space.
+  // range 32-127 is ASCII; 127-136 will be multi-byte UTF-8
```

Comment:
This comment seems like it would be more useful where the 32/137 now appear.

Reply:
moved.